### PR TITLE
Include original data for calculations 

### LINKED ISSUE #2176: Table 2c/2d default data
@minotogna 
The second table of table 2c/2d is a new feature in this cycle, and data auto-calculated (in default) appears to be weired. 

I have re-entered the data which after the update seems to be right;

e.g.2c:

2d:

### LINKED ISSUE #2200: Client side calculations should use odp (see ALB /assessments/fra/2020/ALB/designatedManagementObjective totalForestArea)

### PR DESCRIPTION
**Pull request by sorja**
Resolves #2200 
Resolves #2176 

https://user-images.githubusercontent.com/5508251/218298483-ed14126e-8117-4208-b470-2cd7418e1ea9.mov

https://user-images.githubusercontent.com/5508251/218298484-811616de-2280-49f2-bd52-f9268d44ada3.mov

### CODE REVIEW

File: src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts [hunk 1]
```diff
@@ -50,16 +51,29 @@ const useTableNames = (props: Props): Array<string> => {
...
 export const useGetTableData = (props: Props) => {
   const { assessmentName, countryIso, cycleName, table } = props
+  const country = useCountry(countryIso)
   const { odp } = table.props
 
   const dispatch = useAppDispatch()
   const tableNames = useTableNames(props)
 
+  // if we are using original data point, and we are not in table 1a extent of forest, we need to merge odp data for calculations
+  const mergeOdp =
```
Comment (minotogna) [inline src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: what if we are in table 1b and we're fetching table 1b? should mergeOdp be false ?

Comment (sorja) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: mergeOdp is only false if we are hiding ODPs

Comment (sorja) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: https://user-images.githubusercontent.com/5508251/218316297-458662ea-11a2-478d-9823-0ca4bdfeca86.mov

we cannot hide odps in 1b like we can in 1a

Comment (minotogna) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: what if country.props?.forestCharacteristics.useOriginalDataPoint is false and table.props.name !== TableNames.extentOfForest and tableNames.includes(TableNames.extentOfForest), then data fetched will include extent of forest with mergeOdp to false ?

Comment (sorja) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: Table 1b:
Case1:
Disable ODPs:
```
mergeOdp false
country.props?.forestCharacteristics.useOriginalDataPoint false
tableNames.includes(TableNames.extentOfForest) true
```

Case2:
Enable ODPs:
```
mergeOdp true
country.props?.forestCharacteristics.useOriginalDataPoint true
tableNames.includes(TableNames.extentOfForest) true
```

This is correct, no?

If country disabled ODPs, the validations and calculations should NOT use ODP data?

Comment (minotogna) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: country.props?.forestCharacteristics.useOriginalDataPoint only applies to forestCharacteristics. all calculations involving 1a must include odp

Comment (sorja) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: So, the correct condition is, we always mergeOdp unless we are in 1b and odps disabled?

`table.props.name !== TableNames.extentOfForest && country.props?.forestCharacteristics.useOriginalDataPoint`

**Comment by minotogna** [comment]
- [ ] To investigate different values at every render for tables 2c/2d

https://user-images.githubusercontent.com/1138156/218324308-91aaa261-5c51-4d95-91ab-2c4db057e551.mov

**Review by minotogna — APPROVED** [review]

**Review by rfontanarosa — APPROVED** [review]

**Pull request merged**